zmon_worker_monitor/zmon_worker/tasks/main.py

```diff
@@ -634,6 +635,13 @@ def empty(v):
     return not bool(v)
 
 
+def urlparse(u):
```

Comment:
this needs to go in ``build_default_context``. And I would rename ``u`` to ``url``